adhocracy4/modules/admin.py

```diff
@@ -26,6 +27,12 @@ class ModuleAdmin(admin.ModelAdmin):
         phase_admin.PhaseInline
     ]
     list_filter = ('project__organisation', 'project')
+    if hasattr(settings, 'A4_BLUEPRINT_TYPES'):
+        list_filter += ('blueprint_type',)
+        readonly_fields = ('blueprint_type',)
+    else:
+        exclude = ['blueprint_type']
+
     list_display = ('__str__', 'name')
```

Comment:
Can you also add the blueprint type to the list display when the settings are there?